specification/playwrighttesting/PlaywrightTesting.AuthManager/main.tsp

```diff
@@ -90,7 +107,11 @@ model Account {
 
   @doc("When enabled, this feature allows the workspace to upload and display test results, including artifacts like traces and screenshots, in the Playwright portal. This enables faster and more efficient troubleshooting.")
   @visibility("read")
-  reporting?: EnablementStatus = EnablementStatus.Disabled;
+  reporting?: EnablementStatus = EnablementStatus.Enabled;
+
+  @doc("When enabled, this feature allows the workspace to use local auth (through service access token) for executing operations.")
+  @visibility("read")
+  localAuth?: EnablementStatus = EnablementStatus.Disabled;
```

Comment:
> When enabled, this feature allows the workspace to use local auth (through service access token) for executing operations.

A read-only property means user cannot set the value. So how does user able to "disable"/"enable" this feature?

---

And, why there is a service-side default value (e.g. `EnablementStatus.Disabled`) to this already read-only property?
A server-side default value is used when user does not define the property in request, and server choose the value. How does this work together with a property user cannot set?

Reply:
There's no strong reason. We thought it to add in API contract to inform that if this is not explicitly set by the user, this param has a default value set by the service, for eg. in case of local auth it's disabled by default. It should be always available in response. In case it's absent, it would be a bug and the default value should still be assumed as per contract.

To resolve this conversation, should we remove these values? Would this make sense anywhere in API public contract/documentation?

Reply:
Ok. I think I get the reason.

Personally, I'd expect when server always have the value in response, it be "required" instead of "optional". However, there would be complications on SDK part about the "required"/"optional" on this, so I don't have strong opinion on the changed. You may test out the .NET SDK and JavaScript SDK first, about this.
And hopefully, changing a property from optional to required may be a non-breaking change in the output-only model (again, this may have complications to SDK as well).

I'll approve the PR and add Approval label (as we got 2 approvals on data-plane part of this). Though if you are not in an urgent, you may keep the PR open after Christmas, to see if any more reviewers have opinion on the PR.

Reply:
Thanks @weidongxu-microsoft !

Actually we've our GA planned in Jan last week so we'd like to get green on all the pending KPIs ASAP. Since there're no blockers we'd like to merge this PR and probably can take any minor work later.

Also, we've another upcoming PR for another data-plane service. For now, we'd like to merge this one.